chips/atmega328p-hal/Cargo.toml

```diff
@@ -8,10 +8,12 @@ edition = "2018"
 rt = ["avr-device/rt"]
 # Package exposes the ADC6 and ADC7 pins (only 32TQFP, 32MLF, 32UFBGA)
 adc-pins = []
+# Compile for the atmega328pb version of the chip
+pb = []
```

Comment:
I quite like the pattern the STM32 crates are using here (as I am also looking into merging even more of the HAL crates, see #94).  That would mean:

```toml
device-selected = []
atmega328p = ["avr-device/atmega328p", "device-selected"]
atmega328pb = ["avr-device/atmega328pb", "device-selected"]
```

```rust
#[cfg(not(feature = "device-selected"))]
compile_error!(
    "This crate requires you to specify your target chip as a feature.

    Please select one of the following

    *   atmega328p
    *   atmega328pb
	"
);

#[cfg(feature = "atmega328p")]
pub use avr_device::atmega328p as pac;
#[cfg(feature = "atmega328pb")]
pub use avr_device::atmega328pb as pac;

#[cfg(feature = "device-selected")]
pub mod ... {}
#[cfg(feature = "device-selected")]
pub mod ... {}
```

Reply:
That's a good way to do it. Thanks for the suggestion.